### PR TITLE
RC for 0.15.0

### PR DESCRIPTION
**Pull request by neelip**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,17 @@
...
+## 0.15.0 - 2020-09-14
+### Added
+- PostgreSQL support to Attached Accounts
+- PostgreSQL support to API
+
 ## 0.14.0 - 2020-07-14
 ### Added
 - Use the setting "azureDatabases.batchSize" to configure the Mongo "DBQuery.shellBatchSize" attribute and the batch size when loading children in the tree view.
 - Create and edit stored procedures in PostgreSQL. NOTE: only applies to PostgreSQL server versions 11 and above.
+### Fixed
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md:14]: This should be in the 0.15.0 section

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-cosmosdb",
-    "version": "0.14.1-alpha",
+    "version": "0.15.0",
```
Comment (EricJizbaMSFT) [inline package.json:3]: You also need to update package-lock.json

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,17 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.15.0 - 2020-09-14
+### Added
+- PostgreSQL support to Attached Accounts
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md:7]: I feel like these aren't quite readable enough. Looking at previous entries and modeling after that can be helpful. For example, a previous one says:

> Attach Cosmos DB accounts by connection string

so we might do:

> Attach PostgreSQL servers by connection string

Another example:

> API support for integration with [App Service extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice)

So we could do:

> PostgreSQL support for integration with [App Service extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice)

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,10 +2,17 @@
...
+- PostgreSQL support to API
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md:8]: If we make the default version 10 (as discussed offline), that should be included in a `### Changed` section

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,10 +2,17 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md:3]: You also need to update the ThirdPartyNotices file. Ping me if you need help with that

File: CHANGELOG.md (hunk 1 above)
Comment (neelip) [reply CHANGELOG.md:14]: thank you, I didn't realize it

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,11 +2,17 @@
...
 
+## 0.15.0 - 2020-09-14
+### Added
+- Attach PostgreSQL servers by connection string
+- PostgreSQL support for integration with App Service extension
+### Fixed
+- [Bugs fixed] (https://github.com/microsoft/vscode-cosmosdb/issues?q=is%3Aopen+is%3Aissue+milestone%3A0.15.0)
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md:10]: You should model this after previous links. It's supposed to reference closed issues, not open ones

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,16 +4,15 @@ All notable changes to the "azure-cosmosdb" extension will be documented in this
 
 ## 0.15.0 - 2020-09-14
 ### Added
-- PostgreSQL support to Attached Accounts
-- PostgreSQL support to API
+- Attach PostgreSQL servers by connection string
+- PostgreSQL support for integration with App Service extension
+### Fixed
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md:9]: Take a close look at whitespace and keep that consistent as well. Sometimes that can make a difference in markdown. For example, we usually do an empty line between each "### Added" and "### Fixed" section. Also, I don't think a link works if there's a space between `[Bugs fixed]` and `(https://...`

**Review by EricJizbaMSFT — APPROVED** [review]

**Pull request merged**